charts/velero/templates/schedule.yaml

```diff
@@ -6,14 +6,25 @@ metadata:
   annotations:
   {{- if $schedule.annotations }}
     {{- toYaml $schedule.annotations | nindent 4 }}
+  {{- end }}
+  {{- if $.Values.customAnnotations }}
+  {{- toYaml $.Values.customAnnotations | nindent 4 }}
   {{- end }}
     "helm.sh/hook": post-install,post-upgrade,post-rollback
     "helm.sh/hook-delete-policy": before-hook-creation
   labels:
+    helm.sh/chart: {{ include "velero.chart" $ }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    app.kubernetes.io/component: backup
+    app.kubernetes.io/part-of: {{ template "velero.name" $ }}
     app.kubernetes.io/name: {{ include "velero.name" $ }}
     app.kubernetes.io/instance: {{ $.Release.Name }}
-    app.kubernetes.io/managed-by: {{ $.Release.Service }}
-    helm.sh/chart: {{ include "velero.chart" $ }}
+    {{- if $.Chart.AppVersion }}
+    app.kubernetes.io/version: {{ $.Chart.AppVersion | quote }}
+    {{- end }}
+    {{- if $.Values.customLabels }}
+    {{ toYaml $.Values.customLabels }}
+    {{- end }}
```

Comment:
Any reason not using `{{- include "velero.selectorLabels" . | indent 4 }}` here?